Migrate.md

```diff
@@ -0,0 +1,77 @@
+How to migrate from 1.x.x
+=========================
+
+Changes in the namespace
+========================
+
+First off, start by renaming your namespaces, what used to be
+
+```ruby
+class MyClass < Enumeration::Base
+```
+
+will now be
+
+```ruby
+class MyClass < Enumerations::Base
+```
+
+Configuration
+=============
+
+Since v2 has a breaking change of default values in the database columns, you have 2 options:
+
+1. Create an initializer that you will configure to have the old naming conventions
+2. Create data migrations and convert your data to be alligned with the new Enumerations
+
+First option might be tempting, but you need to weigh in the advantages of the new enumerations.
+With the second option, you can get enumerations in the database instead of ID's, which may prove quite usefull.
+
+First way
+=========
+
+You need to create the following initializer
+
+```ruby
+# config/initializers/enumerations.rb
+
+Enumerations.configure do |config|
+  config.primary_key        = :id
+  config.foreign_key_suffix = :id
+end
+```
+
+And voilà! You can carry on with work knowing your enumerations are up to date.
+
+Second way
+==========
+
+You can use a gem for the migration, or just write raw SQL.
+
+Your migration will look something like this:
+
+```ruby
+class MoveMyEnumToNewEnumerations < ActiveRecord::Migration
+  def change
+    rename_column :my_table, :my_enum_id, :my_enum
+    change_column :my_table, :my_enum, :string
+  end
+
+  def data
```

Comment:
either merge this with change, or write as two migrations. Users wont know what data is.